constants.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+// Константы для маршрутов
+const (
+	RouteNextDate = "/api/nextdate"
+	RouteTask     = "/api/task"
+	RouteTasks    = "/api/tasks"
+	RouteTaskDone = "/api/task/done"
+	RouteSignIn   = "/api/signin"
+)
+
+// Константы ошибок
+const (
+	ErrorMethodNotAllowed  = "Метод не поддерживается"
+	ErrorInvalidID         = "Не указан идентификатор задачи"
```

Comment:
Для того, чтобы работаь с этим переменными как с ошибками, то лучше делать так
```suggestion
	var ErrorInvalidID         = errors.New("Не указан идентификатор задачи")
```